### ISSUE TITLE
no-multiple-empty-lines reports invalid column

### ISSUE THREAD

**Issue by Arcanemagus** [eslint/eslint#7086]
**What version of ESLint are you using?**

`eslint@3.4.0`

**What parser (default, Babel-ESLint, etc.) are you using?**

The default parser

**Please show your full configuration:**

_.eslintrc.json_:

```json
{
  "rules": {
    "no-multiple-empty-lines": 2
  }
}
```

**What did you do? Please include the actual source code causing the issue.**

Ran ESLint on the following code:

```js
'use strict';




```

Note, files used in this test are available here: [linter-eslint_GH687.zip](https://github.com/eslint/eslint/files/462664/linter-eslint_GH687.zip)

**What did you expect to happen?**

An error reported on line 4 (5?), column 1.

**What actually happened? Please include the actual, raw output from ESLint.**

```ShellSession
PS C:\Temp\github\linter-eslint_GH687> .\node_modules\.bin\eslint .\foo.js

C:\Temp\github\linter-eslint_GH687\foo.js
  4:2  error  More than 2 blank lines not allowed  no-multiple-empty-lines

✖ 1 problem (1 error, 0 warnings)
```

Note the column of `2`, which doesn't exist.

I tested this on `eslint@1.10.3`, `eslint@2.0.0`, and `eslint@2.13.1` as well as the reported `eslint@3.4.0` with identical results.

Moving all the way down to `eslint@0.9.0`, where this rule was introduced, results in the following:

```ShellSession
PS C:\Temp\github\linter-eslint_GH687> .\node_modules\.bin\eslint .\foo.js

.\foo.js
  1:0  error  Use the function form of "use strict"  global-strict
  1:0  error  Strings must use doublequote           quotes
  4:1  error  Unexpected blank line at end of file   eol-last
  5:0  error  Multiple blank lines not allowed       no-multiple-empty-lines

✖ 4 problems
```

So I'm guessing that something broke when updating this rule for the 1-based indexing for `eslint@1`.

If there is any further information necessary please let me know! Originally reported by several people over in https://github.com/AtomLinter/linter-eslint/issues/687, although I'm not sure why it didn't show up earlier as this _should_ have been breaking for a _very_ long time there.

**Comment by IanVS** [comment]
Looks like the column location changed in https://github.com/eslint/eslint/commit/13fd818ae8d065a3a2ee7a54a47105bf8f066454#diff-0f6472c36e2e4d88a28a92f9b5148206R85.  It seems that it should be `0` instead of `1` there.
